packages/webapp/src/containers/Finances/useTransactions.js

```diff
@@ -44,7 +44,11 @@ const buildLabourTransactionsFromTasks = ({
   users,
```

Comment:
This file is important too, I updated the functions so they would work well with the actual filter which is an object rather than an array